test/internal/court_data_adaptor_client_test.py

```diff
@@ -1,84 +1,86 @@
+from unittest.mock import PropertyMock, patch, Mock
+
 from laa_court_data_api_app.internal import court_data_adaptor_client
 from test.internal.internal_fixtures import *
 
 
-async def get_token_async():
+def get_token_async():
```

Comment:
as it is something we have to do as apart of test setup ive chosen to keep it as apart of the test so that its clear we can pass this through for future test writing